packages/core/lib/index.ts

```diff
@@ -1,8 +1,21 @@
+/* eslint no-redeclare: "off" */
+import { Logger as _Logger } from '@comunica/types';
+
+// Remove this export, and move test suite to \@comunica/types in the next release
```

Comment:
```suggestion
// TODO: Remove this export, and move test suite to \@comunica/types in the next release
```

Just so that it pops up in my editor as task :-)